formatter/eslintrc_base.json

```diff
@@ -0,0 +1,40 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    // TODO(#3361): enable
+    // "plugin:@typescript-eslint/strict",
+    "google",
+    "prettier"
+  ],
+  "env": {
+    "browser": true,
+    "commonjs": true
+  },
+  "plugins": ["@typescript-eslint"],
+  "rules": {
+    "require-jsdoc": "off",
+    "valid-jsdoc": "warn",
+    "@typescript-eslint/no-unused-vars": "off", // overlaps with standard eslint no-unused-vars.
+
+    // TODO(#3361): fix and reenable
```

Comment:
Which plugin is this TODO for?

Reply:
All of them. I'm planning to reenable them one by one.